### PR TITLE
BazaarLookup Small Edit

### PR DESCRIPTION
**Pull request by VeritasDL**
Added support for AH searching if the item exists on AH instead of BZ

### CODE REVIEW

**Review by kevinthegreat1 — COMMENTED** [review]
Looks good, just need a name now and maybe move it outside of the bazaar config section.

File: src/main/resources/assets/skyblocker/lang/en_us.json [hunk 1]
```diff
@@ -315,7 +315,7 @@
   "skyblocker.config.helpers.bazaar.enableBazaarHelper.@Tooltip": "Draws icons on top of orders to explain the current state of the order.\n\n%s: Order is going to expire soon\n%s: Order has expired\n%s: Order is filled to some degree and there are items/coins to claim\n%s: Order is filled",
   "skyblocker.config.helpers.bazaar.enableBazaarLookup": "Enable Bazaar Lookup",
   "skyblocker.config.helpers.bazaar.enableBazaarLookup.@Tooltip": "Searches the Bazaar for the current hovered item with the F6 key.",
```
Comment (viciscat) [inline src/main/resources/assets/skyblocker/lang/en_us.json:317]: update these 2 lines to inform that it searches bazaar. Add to the tooltip that you need cookie buff

**Review by viciscat — CHANGES_REQUESTED** [review]

**Comment by VeritasDL** [comment]
Bazaar/AH lookup 
https://github.com/SkyblockerMod/Skyblocker/blob/a1c379c5dcc600166a8044e050a10d20dc6759eb/src/main/java/de/hysky/skyblocker/skyblock/bazaar/BazaarHelper.java#L111
things like, pets, enchanted books, and potions return bad names 

Sugar Rush 1 Enchanted Book
```
stack = {ItemStack@31560} "1 minecraft:enchanted_book"
 count = 1
 item = {EnchantedBookItem@31590} "minecraft:enchanted_book"
  registryEntry = {RegistryEntry$Reference@31599} "Reference{ResourceKey[minecraft:item / minecraft:enchanted_book]=minecraft:enchanted_book}"
  components = //trimmed
  translationKey = "item.minecraft.enchanted_book"
  fabric_cachedItemVariant = {ItemVariantImpl@31603} "ItemVariant{item=minecraft:enchanted_book, components={}}"
 components = //trimmed
 baseComponents = //trimmed
  changedComponents = {Reference2ObjectArrayMap@31608}  size = 4
   {ComponentType$Builder$SimpleDataComponentType@31616} "minecraft:custom_name" -> {Optional@31617} "Optional[empty[siblings=[literal{Enchanted Book}[style={color=white,!bold,!italic,!underlined,!strikethrough,!obfuscated}]]]]"
    key = {ComponentType$Builder$SimpleDataComponentType@31622} "minecraft:custom_data"
    value = {Optional@31623} "Optional[{id:"SUGAR_RUSH;1"}]"
     value = {NbtComponent@31634} "{id:"SUGAR_RUSH;1"}"
 skyblockId = null
 skyblockApiId = null
 neuName = null
```
ends up searching for "Enchanted Book"

Level 1 Armadillo
```
item = {PlayerHeadItem@32260} "minecraft:player_head"
 fabric_cachedItemVariant = {ItemVariantImpl@32276} "ItemVariant{item=minecraft:player_head, components={}}"
 changedComponents = {Reference2ObjectArrayMap@32271}  size = 5
  {ComponentType$Builder$SimpleDataComponentType@31616} "minecraft:custom_name" -> {Optional@32291} "Optional[empty[siblings=[literal{[Lvl 1 ➡ 100] }
  key = {ComponentType$Builder$SimpleDataComponentType@31616} "minecraft:custom_name"
   value = {Optional@32291} "Optional[empty[siblings=[literal{[Lvl 1 ➡ 100] }
   value = {MutableText@32321} "empty[siblings=[literal{[Lvl 1 ➡ 100] 
	content = {PlainTextContent$1@32943} "empty"
     siblings = {ArrayList@32944}  size = 2
      0 = {MutableText@32965} "literal{[Lvl 1 ➡ 100] }[style={color=gray,!bold,!italic,!underlined,!strikethrough,!obfuscated}]"
       content = {PlainTextContent$Literal@32969} "literal{[Lvl 1 ➡ 100] }"
      1 = {MutableText@32966} "literal{Armadillo}[style={color=white,!bold,!italic,!underlined,!strikethrough,!obfuscated}]"
       content = {PlainTextContent$Literal@32974} "literal{Armadillo}"
  {ComponentType$Builder$SimpleDataComponentType@31622} "minecraft:custom_data" -> {Optional@32296} "Optional[{id:"ARMADILLO;0"}]"
   key = {ComponentType$Builder$SimpleDataComponentType@31622} "minecraft:custom_data"
   value = {Optional@32296} "Optional[{id:"ARMADILLO;0"}]"
skyblockId = null
skyblockApiId = null
neuName = null
```
ends up searching for  "[Lvl 1 ➡ 100] Armadillo"

**Comment by kevinthegreat1** [comment]
The enchantment book should be fixed already. For the pet problem, I think we'll need to change `ItemStackBuilder#petData` or do some special handling with pet item names. I'm leaning towards the latter cause I don't know how I would change `ItemStackBuilder#petData` since the item stack is displayed and showing pet levels like that makes the most sense.

**Review by kevinthegreat1 — APPROVED** [review]

**Review by kevinthegreat1 — APPROVED** [review]

**Pull request merged**